src/index.js

```diff
@@ -140,7 +138,7 @@ module.exports = {
           this.logger.warn(err)
           return
         }
-        this.logger.error(err)
+        this.logger.error(err, ws.meta)
```

Comment:
Добавь, пожалуйста, тут и выше уточение, что ошибка с moleculer-ws идет. Будет намного проще в кибане распознать откуда ошибки идут
```suggestion
        this.logger.error("moleculer-ws error:", err, ws.meta)
```